README.md

```diff
@@ -73,7 +72,7 @@ brew install tlrc
 Then you have direct access to simplified, easy-to-read help for commands, such as `tar`,
 accessible through typing `tldr tar` instead of the standard `man tar`.
 
-If you want an offline version without installing any software,
+If you don't want to install any software,
```

Comment:
However, I would like to note that I prefer this wording over the old "If you want an offline version without installing any software,"

Reply:
I'm not sure what do you mean - the suggestion was about the web client, not PDFs.

Reply:
Oh, I thought it was about the offline client, overlooked it